decidim-core/lib/decidim/core/test/shared_examples/resource_endorsed_event_examples.rb

```diff
@@ -9,6 +9,7 @@
   let(:author) { create :user, organization: resource.organization }
 
   let(:extra) { { endorser_id: author.id } }
+  let(:resource_title) { decidim_sanitize_translated(resource.title) }
```

Comment:
This change isn't done in `develop`

Reply:
reverted in [bde61ef](https://github.com/decidim/decidim/pull/12547/commits/bde61efea5e7275a335ef994faa30ba21f9aff15)